language/src/ceylon/language/Set.ceylon

```diff
@@ -183,7 +183,7 @@ shared interface Set<out Element=Object>
  This is an eager operation and the resulting set does
  not reflect changes to the given [[stream]]."
 see(`value Iterable.distinct`)
-shared Set<Element> set<Element>(
+shared Set<Element> set<out Element>(
```

Comment:
Like I said before: you know Ceylon better than me. I had _no clue_ that this works.